README.md

```diff
@@ -29,15 +29,20 @@ Why?
 
 ## Usage
 
-To clone the repository and run this website on your local machine, [install Jekyll](https://jekyllrb.com/docs/installation/) for your OS and type the following commands:
+### Dependencies
 
-    $ sudo gem install jekyll bundler
+Install [asdf-vm](https://asdf-vm.github.io/asdf/#/core-manage-asdf-vm) for your OS and now using asdf-vm, install [ruby](https://github.com/asdf-vm/asdf-ruby).
```

Comment:
Better to break them into 2 points. And since the 2nd point is dependent on the first, number them.

```
Dependencies:
1. asdf-vm(for installing ruby easily)
2. ruby
```

But this way we are forcing the developer to install ruby via asdf. I am not sure if that is a good thing or not. Personally, I agree that using scripts makes the installation easier, but these scripts tend to break a lot, and thus prefer a more extensive manual installation.

Reply:
@frextrite don't know much about that, i was just going according to https://github.com/coala/projects/pull/700#discussion_r252567373